src/Share/share.vala

```diff
@@ -31,7 +31,7 @@ public class FeedReader.Share : GLib.Object {
 	private Share()
 	{
 		var engine = Peas.Engine.get_default();
-		engine.add_search_path(Constants.INSTALL_PREFIX + "/" + Constants.INSTALL_LIBDIR + "/pluginsShare/", null);
+		engine.add_search_path(Constants.INSTALL_LIBDIR + "/pluginsShare/", null);
```

Comment:
Maybe we should use https://valadoc.org/glib-2.0/GLib.Path.build_filename.html to avoid issues like this one https://github.com/jangernert/FeedReader/issues/627